docker-compose/db/postgres-setup.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE identityvault;
```

Comment:
It's seems invalid to have dash "-" on the postgresql database name.
See https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
Only letters, underscores, digits (0-9), or dollar signs ($) is allowed.